code/controllers/subsystem/voting/poll_types.dm

```diff
@@ -81,8 +81,8 @@
 
 	minimum_win_percentage = 0.501
 
-	cooldown = 30 MINUTES
-	next_vote = 90 MINUTES //Minimum round length before it can be called for the first time
+	cooldown = 10 MINUTES
+	next_vote = 0 MINUTES //Minimum round length before it can be called for the first time
```

Comment:
```suggestion
	cooldown = 30 MINUTES
	next_vote = 100 MINUTES //Minimum round length before it can be called for the first time
```
так лучше будет